microbench/src/main/java/alluxio/RpcUtilsBench.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
does it make sense for all the microbench java files to be under microbench/test/ and have no files under microbench/src/? although no other modules reference this, having it under test/ instead of src/ is another layer of protection to prevent it from being compiled into a source jar. wdyt?

Reply:
Not sure how this can be done. I tried to move the benchmark to the test sources dir, but then executing the built jar file gave me an error. 

I excluded this module and JMH artifacts from the shaded client and the assembly modules. This should add more protection.

Reply:
bah okay, thanks for giving it a shot and adding the exclusions